aichatweb-aspire.AppHost/Program.cs

```diff
@@ -0,0 +1,30 @@
+var builder = DistributedApplication.CreateBuilder(args);
+
+var qdrantApiKey = builder.AddParameter("qdrantApiKey", secret: true);
+
+var ollama = builder.AddOllama("ollama")
+    .WithDataVolume();
+var chat = ollama.AddModel("chat", "llama3.2");
+var embeddings = ollama.AddModel("embeddings", "all-minilm");
+
+var vectorDB = builder.AddQdrant("vectordb", apiKey: qdrantApiKey, grpcPort: 6334, httpPort: 6333)
+    .WithDataBindMount("./qdrant_data")
+    .WithLifetime(ContainerLifetime.Persistent);
+
+var ingestionCache = builder.AddSqlite("ingestionCache")
+    .WithSqliteWeb();
+
+var webApp = builder.AddProject<Projects.aichatweb_aspire_Web>("aichatweb-app");
+webApp
+    .WithReference(chat)
+    .WithReference(embeddings)
+    .WaitFor(chat)
+    .WaitFor(embeddings);
+webApp
+    .WithReference(vectorDB)
+    .WaitFor(vectorDB);
+webApp
+    .WithReference(ingestionCache)
+    .WaitFor(ingestionCache);
```

Comment:
No need to assign to a variable and create blocks here.

```suggestion
builder.AddProject<Projects.aichatweb_aspire_Web>("aichatweb-app")
    .WithReference(chat)
    .WaitFor(chat)
    .WithReference(embeddings)
    .WaitFor(embeddings);
    .WithReference(vectorDB)
    .WaitFor(vectorDB);
    .WithReference(ingestionCache)
    .WaitFor(ingestionCache);
```